lib/morpheus.gd

```diff
@@ -586,3 +586,24 @@ DeclareOperation("GQuotients",[IsGroup,IsGroup]);
 DeclareOperation("IsomorphicSubgroups",[IsGroup,IsGroup]);
 
 DeclareGlobalFunction("PatheticIsomorphism");
+
+############################################################################
+##
+#F RepresentativesOuterAutomorphismGroup(<grp>, [,<aut>])
+##
+## <#GAPDoc Label="RepresentativesOuterAutomorphismGroup">
+## <ManSection>
+## <Func Name="RepresentativesOuterAutomorphismGroup" 
+## Arg='grp, [,aut]'\>
+## 
+## <Description>
+## This function returns representatives for each element of the outer 
+## automorphism group of the finite group <A>grp</A>.<P/>
+## The optional argument <A>aut</A>, if present, must be the automorphism
+## group of <A>grp</A>. In this case, the function does not need to 
+## comupute the automorphism group of <A>grp</A>. Note that it is
+## not checked whether <A>aut</A> is indeed the automorphism group
+## of <A>grp</A>. <P/>
+## Note: This Algorithm might become slow if the automorphism group
+## of <A>grp</A> is large.
+DeclareGlobalName("RepresentativesOuterAutomorphismGroup");
```

Comment:
```suggestion
DeclareGlobalName("RepresentativesOuterAutomorphismGroupViaRandomSearch");
```
I'd specify the way we search for the representatives. Why? I feel without the suffix `ViaRandomSearch` the name of the function suggests that it is *the* GAP function to compute representatives of the elements of the outer automorphism group. But the function (at least in its current form) is rather specialized, i.e. only applicable where the outer automorphism group is really small.

Reply:
I updated this for now.

Reply:
We added a TODO for now pointing to Alex' suggestion below.